docs/index.md

```diff
@@ -1,128 +1,71 @@
 
-!!! alert alert-success "This documentation covers the latest release: PMM {{release}}"
+!!! alert alert-success "This is the technical documentation for the latest release: [PMM {{release}}](release-notes/{{release}}.md)"
 
 ## What is *Percona Monitoring and Management*?
 
-PMM is a free and open-source solution
-that you can run in your own environment
-for maximum security and reliability.
-It provides thorough time-based analysis for MySQL, PostgreSQL and MongoDB servers
-to ensure that your data works as efficiently as possible.
+PMM is software that helps you improve the performance of database instances, simplify their management, and strengthen their security. With it, you can:
 
-## Architecture
+- Visualize a wide range of out-of-the-box system performance details
 
-The PMM platform is based on a client-server model that enables scalability. It includes the following modules:
+- Collect and analyze data across complex multi-vendor system topologies
 
-* [PMM Client](#pmm-client) installed on every database host that you want to monitor. It collects server metrics, general system metrics, and Query Analytics data for a complete performance overview.
+- Drill-down and discover the cause of inefficiencies
 
-* [PMM Server](#pmm-server) is the central part of PMM that aggregates collected data and presents it in the form of tables, dashboards, and graphs in a web interface.
+- Anticipate performance issues, troubleshoot existing ones
 
-* [Percona Platform](#percona-platform) provides value-added services for PMM.
+- Watch for potential security issues and remedy them
 
-![image](_images/diagram.pmm.client-server-platform.png)
+PMM is efficient, quick to set up and easy to use. It runs in cloud, on-prem, or across hybrid platforms. It is supported by [Percona's legendary expertise](https://www.percona.com/services) in open source databases, and by a [vibrant developer and user community](https://www.percona.com/forums/questions-discussions/percona-monitoring-and-management).
 
-The modules are packaged for easy installation and usage. It is assumed that the user should not need to understand what are the exact tools that make up each module and how they interact. However, if you want to leverage the full potential of PMM, the internal structure is important.
+!!! alert alert-info "Try the demonstration instance of PMM at <https://pmmdemo.percona.com/>"
 
-PMM is a collection of tools designed to seamlessly work together.  Some are developed by Percona and some are third-party open-source tools.
+## How it works
 
-!!! alert alert-info "Note"
-    The overall client-server model is not likely to change, but the set of tools that make up each component may evolve with the product.
+PMM is a client/server application built by Percona with their own and third-party open-source tools.
```

Comment:
why we writing like percona is some other organization?  Why now "OUR own"?

Reply:
Leaking of tone from existing material. It will evaporate over time. But completely agree "built by us with our own ..." sounds better.